src/reader-activation-auth/auth-form.js

```diff
@@ -97,6 +97,32 @@ window.newspackRAS.push( function ( readerActivation ) {
 			};
 			container.setFormAction( 'signin' );
 
+			/**
+			 * Get the redirect URL.
+			 */
+			const getRedirectUrl = () => {
```

Comment:
```suggestion
			const getCheckoutRedirectUrl = () => {
```

We can be more explicit that this redirect URL is meant for checkout redirection.

Also, if you like the idea of the `checkout.js` file, it looks like a nice home for it as well.

Reply:
Great idea! Moved in https://github.com/Automattic/newspack-plugin/pull/3343/commits/1ce3631b59bb4f3288484ec82641665631927b50